src/main/java/net/clgd/ccemux/init/Launcher.java

```diff
@@ -55,41 +64,61 @@ private static void printHelp() {
 	}
 
 	public static void main(String args[]) {
-		new Launcher(args).launch();
-		System.exit(0);
-	}
-
-	private final CommandLine cli;
-	private final Path dataDir;
-
-	private Launcher(String args[]) {
 		// parse cli options
-		CommandLine _cli = null;
+		CommandLine cli;
 		try {
-			_cli = new DefaultParser().parse(opts, args);
+			cli = new DefaultParser().parse(opts, args);
 		} catch (ParseException e) {
 			System.err.println(e.getLocalizedMessage());
 			printHelp();
 			System.exit(1);
+			return;
```

Comment:
Is there a reason that a `return` was added here, but not after line 80?

Reply:
It's needed in order for `cli` to be considered assigned to. But yep, I can add it in both places.